crates/tasks/src/lib.rs

```diff
@@ -45,6 +46,12 @@ pub mod shutdown;
 #[cfg(feature = "rayon")]
 pub mod pool;
 
+/// The global task manager. Required to initialize the task executor.
+static MANAGER: Lazy<Mutex<Option<TaskManager>>> =
+    Lazy::new(|| Mutex::new(Some(TaskManager::new(Handle::current()))));
```

Comment:
this is not what we want here, this type ins intended to be spawned. instead we want to away to retrieve just the executor, with the prerequisite that a `Task manager` exists